src/extension/popups/spend.tsx

```diff
@@ -103,7 +108,7 @@ const ConnectSpend: React.FC = () => {
             {spendParameters.addressRecipients.map((recipient, index) => (
               <div key={index}>
                 <div className="container flex justify-between mt-6">
-                  <span className="text-lg font-medium">{fromSatoshi(recipient.value)}</span>
+                  <span className="text-lg font-medium">{fromSatoshi(recipient.value, getPrecision(recipient.asset))}</span>
```

Comment:
To remove prettier errors

```suggestion
                  <span className="text-lg font-medium">
                    {(fromSatoshi(recipient.value), getPrecision(recipient.asset))}
                  </span>
```